index.js

```diff
@@ -44,21 +43,20 @@ prog
         }
 
         return getLocalDependencies()
-            .then((deps) => clone(deps, logger))
-            .catch((err) => logger.error(LOG.ERROR, err));
+            .then(deps => clone(deps, logger))
+            .catch(err => logger.error(LOG.ERROR, err));
     })
-
     .command('validate', 'Validates local .zel file.')
     .action((args, options, logger) => {
         getLocalDependencies()
-            .then((deps) => {
+            .then(deps => {
                 resolver
-                    .on('valid', (repo) => logger.info(LOG.VALID, repo.repoName))
-                    .on('invalid', (repo) => logger.error(LOG.INVALID, repo.repoName))
+                    .on('valid', repo => logger.info(LOG.VALID, repo.repoName))
+                    .on('invalid', repo => logger.error(LOG.INVALID, repo.repoName))
                     .validate(deps)
-                        .catch((err) => logger.error(LOG.ERROR, err));
+                    .catch(err => logger.error(LOG.ERROR, err));
```

Comment:
May be a personal preference but I do like the extra indentation for `.validate()` here since `validate()` returns a `Promise` so it's more apparent that we're not calling `.catch()` on the `resolver` instance.

Aside from that, aligning these chained methods does look a lot cleaner.